src/planscape/plan/views.py

```diff
@@ -821,18 +822,23 @@ def get_scores(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("failed score fetch: " + str(e))
 
 
-# TODO: finalize call logic after testing piping.
-# NOTE: To send a queue message from your local machine, populate AWS credential args.
-# TODO: feed plan_id and scenario_id from user input into the queue message body 
-def queue_forsys_call(request: HttpRequest) -> HttpResponse:
+# NOTE: To send a queue message from your local machine, populate AWS credentials.
+# TODO: Add tests that mock SQS calls
+def queue_forsys_lambda_prototype(request: HttpRequest) -> HttpResponse:
     try:
         user = get_user(request)
+        scenario_id = create_scenario(request)
```

Comment:
Should scenario status be set to pending at this step?

Reply:
i need to instantiate ForsysGenerationRequestParamsFromDb after this is all done, which is why I didn't update the status. During _validate_scenario, it checks that the status is either initialized or failed. is that ok?